### ISSUE TITLE
Tab completions for PowerShell

### ISSUE THREAD

**Issue by ajeetdsouza** [ajeetdsouza/zoxide#344]
### Discussed in https://github.com/ajeetdsouza/zoxide/discussions/343

<div type='discussions-op-text'>

<sup>Originally posted by **@charlesroper** January 31, 2022</sup>
I've just installed Zoxide for PowerShell (7.2.1) and just wondering: are there no tab completions? Thanks!</div>

**Comment by charlesroper** [comment]
Thanks for considering this - much appreciated 👍

**Comment by boardkeystown** [comment]
Yes I must say zoxide need some powershell love. It's very cool. If I could figure out how to query the zoxiod db it maybe possible to write a ps module that when you press tab provides a list of options or the most likely. Powershell is plays by different rules compared to bash but it should be possible

**Comment by nickcox** [comment]
@boardkeystown: I've been using [@twobiers PR ](https://github.com/ajeetdsouza/zoxide/pull/419)linked above for PowerShell completions and it works pretty well.

Regarding querying the zoxide db, that's what `zoxide query` does, isn't it?

**Comment by AtifChy** [comment]
> @boardkeystown: I've been using [@twobiers PR ](https://github.com/ajeetdsouza/zoxide/pull/419)linked above for PowerShell completions and it works pretty well.
> 
> Regarding querying the zoxide db, that's what `zoxide query` does, isn't it?

any idea why it was rejected?

**Comment by michalczerwinski** [comment]
The command you need is simple:
```pwsh
Register-ArgumentCompleter -CommandName z -Native -ScriptBlock { param($stringMatch) zoxide query $stringMatch --list}
```

@ajeetdsouza will you accept PR for this?
